service/src/main/java/tds/session/repositories/impl/ExternalSessionConfigurationRepositoryImpl.java

```diff
@@ -25,15 +25,24 @@ public ExternalSessionConfigurationRepositoryImpl(final DataSource dataSource) {
         final SqlParameterSource parameters = new MapSqlParameterSource("clientName", clientName);
 
         String query = "SELECT " +
-            "clientname as clientName, \n" +
-            "environment \n" +
+            "externs.clientname as clientName, \n" +
+            "externs.environment, \n" +
+            "_externs.shiftwindowstart, \n" +
+            "_externs.shiftwindowend \n" +
             "FROM session.externs \n" +
-            "WHERE clientname = :clientName";
+            "JOIN session._externs ON _externs.clientName = externs.clientName \n" +
```

Comment:
Actually, I think there are issues with H2 (yet again) and accessing it without the schema name.   I'm sure there is a fix for it but this works and didn't require me to investigate H2 so I skipped it.  This happened to me earlier and I forgot about it until I tried to remove them today.